app/assets/javascripts/discourse/app/controllers/invites-show.js

```diff
@@ -45,18 +50,26 @@ export default Controller.extend(
     },
 
     @discourseComputed(
```

Comment:
you can replace all of this by using or()

Reply:
Excellent suggestion! Made it so.